cumulusci/tasks/push/tasks.py

```diff
@@ -231,13 +237,31 @@ def _init_options(self, kwargs):
 
         # Set the namespace option to the value from cumulusci.yml if not
         # already set
+        if "orgs" not in self.options and "csv" not in self.options:
+            raise TaskOptionsError(
+                "Please call this task with the orgs or csv option. Both of these options require a file name."
+            )
         if "namespace" not in self.options:
             self.options["namespace"] = self.project_config.project__package__namespace
         if "batch_size" not in self.options:
             self.options["batch_size"] = 200
+        if "csv" not in self.options and "csv_field_name" in self.options:
+            raise TaskOptionsError("Please provide a csv file for this task to run.")
+        if (
+            "csv" in self.options
+            or "orgs" in self.options
```

Comment:
I think we only care about `csv_field_name` when `csv` is passed, since `orgs` can only have one org ID per line.